### PR TITLE
cooy of trials project

### PR DESCRIPTION
**Pull request by aizjicod**
in this project I did the following steps:

- copied the project I did on the trials of microverse

### CODE REVIEW

**Review by aizjicod — COMMENTED** [review]
`h1{
    font-size: 50px;
    margin-bottom: 0;
}
#description{
    margin-top:0;
    margin-bottom:3em;
    text-align: center;
}
h3{
    font-weight:bolder;
    font-size: 30px;
}
`

kindly try not to jump from different numbers of 'h'
hence try to go in order :)

**Review by OmarRamoun — CHANGES_REQUESTED** [review]
# 🟡 Just One More Thing 🔸

## Hi @aizjicod 👋

Great work on making the changes requested ♻️ by a previous reviewer 👏.

You've done well implementing some of the requested changes 💯.

Though, There are some **few** issues 🐛 that you still need to work on to go to the next project but you are almost there! 🏃

## Don't Feel frustrated 🤔

I am sure you worked hard 🛠️ to make these changes and actually you have to be proud of them 🥇.

And I understand how much you wanted to get this project approved ✔️ (I was at your place one day 😉). But, don't worry, You still can do it! 💪.

Just apply the changes ♻️ down below 👇 and this time it is going to be approved ✅. Just make sure that you **don't miss any of the changes** requested or introduce any **new bugs** 🐛.

> **Note:** A good tip is to check the box next to every change after you implement it ☑️. And you can also check the full website to see if it is still working after you have implemented the changes.

*Please, reach to [me](#thank-you-for-your-time-🙏-and-call-me-if-you-need-help-or-clarification-☎️) for any help 🪝 or any question ❓. I am always happy to help you 😃.*

## Required Changes ♻️

- [ ] [REQUIRED] Please, use inline comments as mentioned in the project requirements and in the instructions video.
- [ ] [REQUIRED] Please, mention at least 2 changes for the student (you have mentioned one) as mentioned in the project requirements 

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.

## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@omarramoun](https://github.com/omarramoun)
- This is My **Twitter**: [@omarramoun](https://twitter.com/omarramoun)
- This is My **Slack**: [Ramoun](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by aizjicod** [comment]
> 

yes, I already did that in the comments below the first comment.

**Comment by aizjicod** [comment]
Yes I already did that on the comments bellow I might be wrong but for me
that is the case?

File: index.html [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>survey form microverse</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <div id="main">
```
Comment (aizjicod) [inline index.html:11]: kindly change div to main. as it is a semantic tag.

**Review by aizjicod — COMMENTED** [review]
kindly make changes on the semantics tags described

File: index.html [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    </div>
+
+    <div class="input">
+    <label for="number" id="number-label">How Old Are You</label>
+    <input type="number" id="number" required min="12" max="99" placeholder="age">
+    </div>
+
+    <div class="input">
+    <label for="movies" id="selector">choose your favorite movie </label>
+  <select type="dropdown" id="dropdown">
```
Comment (aizjicod) [inline index.html:37]: indentation error.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    <option value="raganrok">thor:raganrok</option>
+    <option value="widow">black widow</option>
+    <option value="doctor">doctor strange</option>
+    <option value="endgame">end game </option>
+    <option value="noway">spiderman: no way home </option>
+  </select>
+  </div>
+
+  <div class="input radials">
+  <h3>How Many Movies Have You Seen?</h3>
```
Comment (aizjicod) [inline index.html:47]: indentation error

File: index.html [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+    <label for="1"><input class="nmov" type="radio" name="nmovies" value="1">1</label>
+    <label for="2"><input class="nmov" type="radio" name="nmovies" value="2">2</label>
+    <label for="3"><input class="nmov" type="radio" name="nmovies" value="3">3</label>
+    <label for="all"><input class="nmov" type="radio" name="nmovies" value="all">All</label>
+  </div>
+
+  <div class="input checkbox">
+  <h3>Favorite Heroe?</h3>
```
Comment (aizjicod) [inline index.html:55]: indentation error.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+  <div class="input">
+    <h3>Comments:</h3>
+    <textarea name="" id="textarea" cols="30" rows="10"></textarea>
+  </div>
+
+
+  <button id="submit">Submit</button>
+</form>
+
+  </div>
```
Comment (aizjicod) [inline index.html:71]: indentation error

**Review by aizjicod — COMMENTED** [review]
kindly try to keep the indentation of the page. it will make the codebase cleaner.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+    <title>survey form microverse</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <div id="main">
+  
+     <header>
+    <h1 id="title">Marvel Universe Survey</h1>
```
Comment (aizjicod) [inline index.html:14]: and h1 tag.

File: index.html (hunk 3 above)
Comment (aizjicod) [inline index.html:47]: and h3 tag.

**Review by aizjicod — COMMENTED** [review]
kindly try not to jump on the h tags numbers. hence try to go in order to make the page look better.

**Review by koachcode — APPROVED** [review]
Hi @aizjicod , 

# STATUS: APPROVED 🟢

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Used inline comment :+1: 
- Used polite sentences :+1: 


#### suggestion 
- please also check linters as well if it is setup correctly or not

well-1

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**